### PR TITLE
Add Forge compatibility

### PR DESCRIPTION
**Pull request by ByThePowerOfScience**
Yeah yeah it's a Fabric mod, but it absolutely works with the Forge version of SVC too (with Sinytra Connector to use Replay on Forge) as long as it's annotated with the annotation the Forge version expects.  

This is specifically for people who play on a Forge server that enforces the Forge version of SVC (and thus can't use the Fabric version), but who _also_ want to use Replay Mod, so they install the Connector and record a ton of replays only to realize that none of them have voice chat and are thus completely useless.  Not that I would have any experience with that. >->

Anyway, this PR just adds the Forge entrypoint to the mod (shading the annotation class to avoid classloading issues on non-Forge versions), making it fully compatible with the Forge version of Simple Voice Chat without _any_ major refactoring or compatibility concerns. :D

### CODE REVIEW

**Comment by ByThePowerOfScience** [comment]
IDK why the fast-forward commit was added to the PR.

File: src/main/java/de/maxhenkel/voicechat/api/ForgeVoicechatPlugin.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package de.maxhenkel.voicechat.api;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+@Retention(RetentionPolicy.RUNTIME)
+public @interface ForgeVoicechatPlugin {
+}
```
Comment (henkelmax) [inline src/main/java/de/maxhenkel/voicechat/api/ForgeVoicechatPlugin.java:8]: Why did you add this?

**Review by henkelmax — CHANGES_REQUESTED** [review]

File: src/main/java/de/maxhenkel/voicechat/api/ForgeVoicechatPlugin.java (hunk 1 above)
Comment (ByThePowerOfScience) [reply src/main/java/de/maxhenkel/voicechat/api/ForgeVoicechatPlugin.java:8]: This keeps the program from crashing on non-Forge versions while doing nothing if the class is already present.

Comment (henkelmax) [reply src/main/java/de/maxhenkel/voicechat/api/ForgeVoicechatPlugin.java:8]: That's not true. This is part of the API.
https://github.com/henkelmax/simple-voice-chat/blob/1.20.6/api/src/main/java/de/maxhenkel/voicechat/api/ForgeVoicechatPlugin.java

Comment (ByThePowerOfScience) [reply src/main/java/de/maxhenkel/voicechat/api/ForgeVoicechatPlugin.java:8]: Oh! In that case, it can be removed safely.

**Comment by henkelmax** [comment]
Thanks!
